### ISSUE TITLE
More intuitive name

### ISSUE THREAD

**Issue by chaonan99** [kudkudak/word-embeddings-benchmarks#54]
Thanks for the great package! I know coming up with a good name is hard, but the short cut "web" used here is a bit confusing and may cause conflictions with other packages such as [web.py](http://webpy.org/).

**Comment by cveres** [comment]
Yes it does!! I have web.py installed and can't for the life of me figure out how to import both. Does anybody know?

**Comment by cveres** [comment]
chaonan99:
Yes it does!! I have web.py installed and can't for the life of me figure out how to import both. Does anybody know?
